public/pages/Indices/containers/IndicesActions/index.tsx

```diff
@@ -46,6 +50,33 @@ export default function IndicesActions(props: IndicesActionsProps) {
     }
   }, [services, coreServices, props.onDelete, onDeleteIndexModalClose]);
 
+  const onCloseFlyout = () => {
+    setSplitIndexFlyoutVisible(false);
+  };
+
+  const splitIndex = async (targetIndex: String, settingsPayload: IndexItem["settings"]) => {
+    const result = await services?.commonService.apiCaller({
+      endpoint: "indices.split",
+      method: "PUT",
+      data: {
+        index: selectedItems.map((item) => item.index).join(","),
+        target: targetIndex,
+        body: {
+          settings: {
+            index: settingsPayload,
+          },
+        },
+      },
+    });
+    if (result && result.ok) {
+      coreServices?.notifications.toasts.addSuccess(`Successfully submit split index request.`);
+      onDelete();
+    } else {
+      coreServices.notifications.toasts.addDanger(result?.error || "");
```

Comment:
It would be better to return if the result.ok is not true, user can modify the split settings and try again, or the flyout will be closed.

Reply:
Good suggestion, will move oncloseflyout inside result.ok condition